presto-product-tests/bin/product-tests-suite-5.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Please update the file with the changes from https://github.com/prestosql/presto/pull/1514